src/pybind/mgr/cephadm/module.py

```diff
@@ -2695,10 +2695,12 @@ def upgrade_ls(self, image: Optional[str], tags: bool, show_all_versions: Option
         return self.upgrade.upgrade_ls(image, tags, show_all_versions)
 
     @handle_orch_error
-    def upgrade_start(self, image: str, version: str, daemon_types: Optional[List[str]] = None, host_placement: Optional[str] = None,
+    def upgrade_start(self, image: str, version: str, do_not_reduce_mds: Optional[bool] = False, daemon_types: Optional[List[str]] = None, host_placement: Optional[str] = None,
                       services: Optional[List[str]] = None, limit: Optional[int] = None) -> str:
         if self.inventory.get_host_with_state("maintenance"):
             raise OrchestratorError("upgrade aborted - you have host(s) in maintenance state")
+        # if not (isinstance(do_not_reduce_mds, bool) or None):
+        #     raise OrchestratorError('do_not_reduce_mds must be a bool value')
```

Comment:
?

Reply:
Leftover code, I'll remove it the next push. Thanks.